modules/services/featureflag/src/main/java/au/com/shiftyjelly/pocketcasts/featureflag/Feature.kt

```diff
@@ -20,4 +20,9 @@ enum class Feature(
         title = "Patron",
         defaultValue = false
     ),
+    AUTO_PLAY_UP_NEXT_SETTING(
+        key = "auto_play_when_up_next_empty",
+        title = "Auto Play When Up Next Empty Setting",
+        defaultValue = BuildConfig.DEBUG,
```

Comment:
The default value is true for debug build, right? Asking because of a contradictory test instruction:

> On debug builds, verify that when the "Auto Play When Up Next Empty Setting" beta toggle is off,

Reply:
That's right, I set it to default to `true` for debug builds. I didn't see any reason to have it off, and that makes it easier to test/catch-issues.